docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -36,6 +36,35 @@ ifeval::["{provider}" == "stackit"]
 `component-openshift4-terraform` doesn't currently support the STACKIT cluster Terraform module, so at this time, the Terraform output for NS records isn't provided.
 --
 endif::[]
+ifeval::["{provider}" == "cloudscale"]
+. Store the subnet UUID in the cluster configuration
++
+[source,bash]
+----
+export SUBNET_UUID="`terraform output -raw subnet_uuid`"
+
+pushd ../../../inventory/classes/${TENANT_ID}
+
+yq eval -i ".parameters.openshift.cloudscale.subnet_uuid = $SUBNET_UUID" \
+  ${CLUSTER_ID}.yml
+
+git commit -am "Add Cloudscale Subnet UUID to cluster configuration for ${CLUSTER_ID}"
+git push
+popd
+popd
+----
+
+. Compile and push the cluster catalog
++
+include::partial$install/commodore-dynfacts.adoc[]
+
+. Return to Terraform directory
++
+[source,bash]
+----
+pushd catalog/manifests/openshift4-terraform/
+----
+endif::[]
```

Comment:
This needs a `terraform init` again, since the catalog directory is wiped and recreated by `commodore catalog compile`.